React-Projects/job-portal/src/App.css

```diff
@@ -0,0 +1,12 @@
+.empty-list-container {
+  text-align: center;
+  .empty-list-img {
+    margin-top: 20px;
+    height: 100px;
```

Comment:
@codal-kcolaco please avoid giving height and weight to images as static, one should be always 'auto'.

Reply:
@abhi-codal ok will make sure from next time